src/controllers/v2/auth.controller.js

```
@@ -0,0 +1,167 @@
+    case "RESTAURANT_OWNER":
+      const { phoneRestaurantOwner, panNumber } = z
+        .object({
+          phoneRestaurantOwner: z
+            .string({ required_error: "Phone number is required." })
+            .length(10, "Phone number must be exactly 10 digits long."),
+          panNumber: z
+            .string({ required_error: "PAN number is required." })
+            .length(10, "PAN number must be exactly 10 characters long.")
+            .regex(/^[A-Z]{5}[0-9]{4}[A-Z]{1}$/, "Invalid PAN number format."),
+        })
+        .parse(req.body);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)